README.md

```diff
@@ -1 +1,52 @@
 # saucelabs-connector
+
+Helps connect the local machine to SauceLabs and starts a browser with the specified page url.
+
+##Install
+
+`$ npm install saucelabs-connector`
+
+##Usage
+```js
+
+var SauceLabsConnector = require('saucelabs-connector');
+
+// Use this online tool to generate a valid platform configuration: 
+// https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/
+var browserInfo = {
+    platform:    'Windows 10',
+    browserName: 'chrome'
+};
+
+var pageUrl = 'www.example.com';
+var jobName = 'Sample tests';
+
+var saucelabsConnector = new SauceLabsConnector('SAUCELABS_USERNAME', 'SAUCELABS_ACCESS_KEY');
+var saucelabsBrowser   = null;
+
+// Connects the local machine to SauceLabs
+saucelabsConnector
+    .connect()
+    .then(function () {
+        // Use the waitForFreeMachines method to ensure that the required number of machines is available.
+        var machinesCount    = 3;     // the amount of needed machines.
+        var requestInterval  = 30000; // the request delay in milliseconds.
+        var maxAttemptsCount = 5;     // the maximum count of attempts.
+        
+        return saucelabsConnector.waitForFreeMachines(machinesCount, requestInterval, maxAttemptsCount);
+    })
+    .then(function () {
+        // Starts a remote browser on SauceLabs with the specified url
+        return saucelabsConnector.startBrowser(browserInfo, pageUrl, jobName);
+    })
+    .then(function (browser) {
+        saucelabsBrowser = browser;
+        // do some work with the browser
+    })
+    .then(function () {
+        // closing the browser
```

Comment:
I don't feel like it annoys me. But let's do it since you've noticed.
BTW, I'd rather prefer Continuous here, but there is a rule in DX to write comments that start with a verb (impersonal) in Simple.